src/pages/overview/pdf-services-api/howtos/webhook-notification.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Webhook Notification support for Adobe PDF Services APIs | Adobe PDF Services
+---
+# **Webhook Notification support for Adobe PDF Services APIs**
+
+Webhooks are now supported in Adobe PDF Service APIs. Webhooks are the plugins which help in getting the notification of job completion. Clients will not be required to invoke the GET Status API to get the job completion status and result, instead, they will be automatically notified with the job status and result.
+
+To use webhooks, client needs to pass **notifiers** key in the request body while invoking the Submit Job API. 
+**notifiers key** expects the list of objects where each object corresponds to one webhook. Each notifier object consists of **type** and **data** fields. Below are the details of these fields and sample request :
+
+1. **type**: Type of notifier needs to be invoked. Currently supported notifier type is [CALLBACK](#callback-notifier).
+2. **data**: This contains the data needed for the webhook. It varies depending on the type of notifier.
+
+### Sample Request Body
+```json
+{
+  "assetID": "{{assetID}}",
+  ....,
+  ///REST API Operation related details
+  ....
+  
+  "notifiers": [
+    {
+      "type": "<NOTIFIER_TYPE>",
+      "data": {
+          //<NOTIFICATION_DATA>
+        }
+      }
+    }
+  ]
+}
+```
+
+<InlineAlert slots="text"/>
+<div>
+Webhook notification support is only available in REST APIs. It is currently <b>not</b> supported in SDKs.
+</div>
+
+## **CALLBACK Webhook**
+The callback webhook allows client to get the job completion notification on the https url. Client needs to create a HTTPS POST URL, and share that URL and related headers information in the request. Once the job completes, we create a notification payload containing the job status and output asset details and invoke the HTTP POST call on the url provided by the client.
```

Comment:
Callback Webhook Usage

The callback webhook feature allows the client to receive notifications regarding job completion on an HTTPS URL. To enable this feature, the client is required to create an HTTPS POST URL and share that URL along with the necessary header information in the request. Once the job has been successfully completed, we generate a notification payload that includes information about the job status and the details of the output assets. This payload is then sent via an HTTP POST request to the URL provided by the client.

Parameters for the Notifier Object

When utilizing the callback webhook, the following parameters need to be included within the notifier object:

type (Required): This field should always be set to "CALLBACK."

data (Required): The "data" field requires the following keys:

url (Required): This is the HTTPS POST URL created by the user.

headers (Optional): This is a map of key-value pairs and represents header-related information required to make the URL call. This field is optional but can be used to provide additional context or authentication for the callback.